osm2streets/src/lanes/classic.rs

```diff
@@ -46,42 +46,60 @@ pub fn get_lane_specs_ltr(tags: &Tags, cfg: &MapConfig) -> Vec<LaneSpec> {
 
     add_sidewalks_and_shoulders(&mut fwd_side, &mut back_side, &tags, cfg);
 
-    LaneSpec::assemble_ltr(fwd_side, back_side, cfg.driving_side)
+    if let Some(x) = tags.get("turn:lanes:forward") {
```

Comment:
Some of these are easier to reason about with the inner-to-outer order, and some in left-to-right order. So we apply these before and after stitching together the two sides